src/main/webapp/calendar.js

```diff
@@ -4,8 +4,13 @@ goog.require('templates.eventsoy');
 
 function initEvents() {
   displayCalendar();
+  addressAutoComplete();
 }
 
+function addressAutoComplete() {
+  const address = document.getElementById('address');
+  let searchBox = new google.maps.places.SearchBox(address);
```

Comment:
this isn't used anywhere else. could it be a const or are you planning on changing it later?